src/main/scala/it/unibo/pps/bunny/util/ExplainedException.scala

```diff
@@ -1,5 +1,9 @@
 package it.unibo.pps.bunny.util
 
-class ExplainedException(exp: String) extends Exception {
-  print(exp + "\n")
+/**
+ * And Exception with and explanation.
```

Comment:
non so se è un errore di battitura, volevi scrivere un'eccezione o anche un'eccezione?
```suggestion
 * An Exception with andexplanation.
```